Dockerfile

```diff
@@ -29,16 +29,15 @@ ENV GOCOVERDIR /test_coverage
 
 COPY ./integ_entrypoint.sh /integ_entrypoint.sh
 COPY . $GOPATH/src/github.com/awslabs/soci-snapshotter
-RUN apk update && apk upgrade
-RUN apk add --no-cache \
-    btrfs-progs-libs \
-    curl \
-    fuse \
-    gcc \
-    libc6-compat \
-    libseccomp-dev \
+RUN dnf update && dnf upgrade && dnf install -y \
```

Comment:
Does go come by default on `amazonlinux:2023` images? Just noticed we're switching from a `golang-alpine` image to a standard AL2023 image, but we're not `dnf` installing golang

Reply:
We don't actually use go inside the container. We build the snapshotter on the host and copy the binary into the container and the integ tests run on the host with execs into the container to run nerdctl commands.